### PR TITLE
📊 oecd: Update Healthcare expenditure and financing database

### PR DESCRIPTION
**Pull request by paarriagadap**
Replaces and expands [the previous version of the dataset](https://owid.cloud/admin/datasets/489), uploaded 6 years ago.

I am waiting for a confirmation on the units of monetary values, but it can be reviewed nevertheless.

https://github.com/owid/owid-issues/issues/809

### CODE REVIEW

File: snapshots/oecd/2024-02-23/health_expenditure.csv.dvc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    title: OECD Health Expenditure and Financing Database
+    description: |-
+      A System of Health Accounts 2011 provides an updated and systematic description of the financial flows related to the consumption of health care goods and services. As demands for information increase and more countries implement and institutionalise health accounts according to the system, the data produced are expected to be more comparable, more detailed and more policy relevant. It builds on the original OECD Manual, published in 2000 to create a single global framework for producing health expenditure accounts that can help track resource flows from sources to uses. It is the result of a collaborative effort between the OECD, WHO and the European Commission, and sets out in more detail the boundaries, the definitions and the concepts – responding to health care systems around the globe – from the simplest to the more complicated. The accounting framework is organised around a tri-axial system for the recording of health care expenditure, namely classifications of the functions of health care (ICHA-HC), health care provision (ICHA-HP), and financing schemes (ICHA-HF).
```
Comment (spoonerf) [inline snapshots/oecd/2024-02-23/health_expenditure.csv.dvc:8]: Maybe just add some spacing in here to break up the text so it's a bit easier to read on the data page.

**Review by spoonerf — APPROVED** [review]
Mega Jinja use! LGTM, just a couple of data page format related comments.

File: etl/steps/data/garden/oecd/2024-02-23/health_expenditure.meta.yml [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+
+      <% elif financing_scheme == "Government schemes" %>
+      The characteristics of government health care financing schemes are determined by law or by the government. A separate budget is set for the programme, and a government unit has an overall responsibility for it. Usually, but not necessarily, government schemes are operated by government unit(s). The government schemes may also be managed by NPISH or by an enterprise.
+
+      Government (health care financing) schemes have the following characteristics:
+      - Mode of participation: automatic for all citizens/residents, or for a specific group of the population (e.g. the poor) defined by law/government regulation;
```
[the remark this answers is not in the document]
Comment (paarriagadap) [reply etl/steps/data/garden/oecd/2024-02-23/health_expenditure.meta.yml:18]: Yeah, it seems to be a markdown rendering problem. I posted in `#data-issues`

**Comment by paarriagadap** [comment]
Thank you @spoonerf! It was fun to use Jinja for the first time

**Pull request merged**